src/status_im2/contexts/emoji_picker/view.cljs

```diff
@@ -98,30 +93,31 @@
     :container-style style/empty-results}])
 
 (defn- render-list
-  [{:keys [theme filtered-data on-viewable-items-changed scroll-enabled on-scroll on-select
-           set-scroll-ref close]}]
-  (let [data (if filtered-data filtered-data emoji-picker.data/flatten-data)]
-    [gesture/flat-list
-     {:ref                             set-scroll-ref
-      :scroll-enabled                  @scroll-enabled
-      :data                            data
-      :initial-num-to-render           20
-      :max-to-render-per-batch         20
-      :render-fn                       render-item
-      :get-item-layout                 get-item-layout
-      :keyboard-dismiss-mode           :on-drag
-      :keyboard-should-persist-taps    :handled
-      :shows-vertical-scroll-indicator false
-      :on-scroll-to-index-failed       identity
-      :empty-component                 [empty-result]
-      :on-scroll                       on-scroll
-      :render-data                     {:close     close
-                                        :theme     theme
-                                        :on-select on-select}
-      :content-container-style         style/list-container
-      :viewability-config              {:item-visible-percent-threshold 100
-                                        :minimum-view-time              200}
-      :on-viewable-items-changed       on-viewable-items-changed}]))
+  [{:keys [theme filtered-data on-viewable-items-changed scroll-enabled on-scroll
+           on-select set-scroll-ref close sheet-animating?]}]
+  [gesture/flat-list
+   {:ref                             set-scroll-ref
+    :scroll-enabled                  @scroll-enabled
+    :data                            (or filtered-data emoji-picker.data/flatten-data)
+    :initial-num-to-render           14
+    :max-to-render-per-batch         10
+    :render-fn                       render-item
+    :get-item-layout                 get-item-layout
+    :keyboard-dismiss-mode           :on-drag
+    :keyboard-should-persist-taps    :handled
+    :shows-vertical-scroll-indicator false
+    :on-scroll-to-index-failed       identity
+    :empty-component                 [empty-result]
+    :on-scroll                       on-scroll
+    :render-data                     {:close     close
+                                      :theme     theme
+                                      :on-select on-select}
+    :content-container-style         style/list-container
+    :viewability-config              {:item-visible-percent-threshold 100
+                                      :minimum-view-time              200}
+    :on-viewable-items-changed       on-viewable-items-changed
+    :remove-clipped-subviews         true
```

Comment:
@ulisesmac - We need to remove the `remove-clipped-subviews` prop as this will cause a blank list bug in `iOS` when the user navigates to any emoji category which is not rendered.

The official docs state to use at our own risk:
> This may improve scroll performance for large lists. On Android the default value is true.
> > Note: May have bugs (missing content) in some circumstances - use at your own risk.

https://reactnative.dev/docs/flatlist#removeclippedsubviews

As the default value in Android is true, and we don't have any issues in iOS scrolling, we can remove this prop.

Reply:
Thanks @smohamedjavid tested it in an iOS simulator but didn't catch it.

I added because in Android it wasn't a problem so I thought on iOS it'd be the same.